deploy/helm/templates/statefulset.yaml

```
@@ -81,16 +81,76 @@ spec:
           startupProbe:
             # The `livenessProbe` and `readinessProbe` will be disabled until the `startupProbe` is successful.
             httpGet:
-              path: /
               port: http
+            {{- if .Values.probes }}
+            {{- if .Values.probes.startupProbe.api | quote }}
+              path: {{ .Values.probes.startupProbe.api | quote }}
+            {{- else }}
+              path: "/api/v1/health"
+            {{- end }}
```

Can we simplify this with the [`default`](https://helm.sh/docs/chart_template_guide/function_list/#default) or [`dig`](https://helm.sh/docs/chart_template_guide/function_list/#dig) functions? It'll be easier to read and reason with I think. What say?
Similarly in other such cases as well.

```suggestion
              path: {{ dig "probes" "startupProbe" "api" "/api/v1/health" .Values | quote }}
```